pkg/schedule/schedulers/scheduler_controller.go

```diff
@@ -491,6 +491,8 @@ retry:
 			// If the evict-leader-scheduler is disabled, it will obstruct the restart operation of tikv by the operator.
 			// Refer: https://docs.pingcap.com/tidb-in-kubernetes/stable/restart-a-tidb-cluster#perform-a-graceful-restart-to-a-single-tikv-pod
 			if labelMgr.ScheduleDisabled(region) && !isEvictLeaderScheduler {
+				log.Debug("region is blocked by labeler", zap.Uint64("region-id", region.GetID()), zap.ByteString("start-key", region.GetStartKey()),
```

Comment:
Do we need to use `logutil.ZapRedactByteString`? cc @JmPotato

Reply:
Yes, please use that to output the sensitive info like the user key.